v4m-container/README.md

```diff
@@ -129,6 +118,20 @@ docker run v4m monitoring/bin/deploy_monitoring_cluster.sh
 ```
 
 ### Updating kubeconfig and user_dir Directories used by Docker Container
+
 If the files in the `./v4m-container/kubeconfig` or `./v4m-container/user_dir` directories, you will need to do one of the following in order for the change to reflect in your Docker container:
```

Comment:
Missing a word or two (maybe "you change") here....: _If you change the files in..._